src/locales/en/translation.json

```diff
@@ -226,7 +227,8 @@
     }
   },
   "home": {
-    "apiOffline": "Our API is offline. We’re trying to reconnect",
+    "apiOffline": "Our {{ target }} API is offline. We’re trying to reconnect",
+    "runtimeOffline": "Our {{ target }} Paratime is offline. Please try again later.",
```

Comment:
I don't think we'll have a way to detect if a Paratime is offline. It'll either be analyzer or indexer being offline

Reply:
Maybe, but in theory, the runtime status API endpoint could have some useful info. I have alreadly opened a slack thread about this.

Reply:
Yes.. I mean we will detect either paratime analyzer or paratime indexer is offline, not paratime itself

Reply:
Right, what we can easily do is checking how much are we behind the current time, and if it's above a configured threshold, we can display a warning saying that a data is not up to date, processing has only done up to this timestamp...

Reply:
OK, we no longer say that the paratime is offline, but that that we don't have fully up-to-date data.

![image](https://github.com/oasisprotocol/explorer/assets/2093792/55dcf061-0ed7-4761-a257-abd7a5e92cc6)